index.d.ts

```diff
@@ -36,6 +36,46 @@ export interface Hooks {
 	afterResponse?: AfterResponseHook[];
 }
 
+export interface RetryOptions {
+	/**
+ 	Number of retries to retry failed requests
+
+ 	@default 2
+
+ 	@example
+ 	```
+ 	import ky from 'ky';
+
+ 	const parsed = await ky('https://example.com', {retry: {retries: 10, methods: ['get'], statusCodes: [413]}}).json();
```

Comment:
Should be wrapped in an async IIFE `(async () => { })();`

It's also not very readable on just one line.

Also, this code example should be added to the option docs in the readme too. The readme and TS definition should be in sync.